gui-tk/minsky.tcl

```diff
@@ -703,6 +703,12 @@ proc cut {} {
     minsky.cut
 }
 
+# Paste item(s) at position x and y on the canvas upon click down. For ticket 1098
+proc pasteAt {} {
+	canvas.mouseDown [get_pointer_x .wiring.canvas] [get_pointer_y .wiring.canvas]
```

Comment:
The problem with this approach is that the mouse is currently over the "Paste" menu item, particularly with the defintion on line 690